iree/compiler/Dialect/HAL/Transforms/PackAllocations.cpp

```diff
@@ -126,15 +126,17 @@ class PackAllocationsPass
 
     Value offset = baseOffset;
     for (auto &slice : slices) {
-      auto sliceSize =
-          IREE::Util::align(loc, slice.dynamicSize, rangeAlignment, builder);
+      auto sliceSize = builder.createOrFold<IREE::Util::AlignOp>(
+          loc, slice.dynamicSize, rangeAlignment);
       slice.packedOffset.replaceAllUsesWith(offset);
-      offset = IREE::Util::align(
-          loc, builder.createOrFold<arith::AddIOp>(loc, offset, sliceSize),
-          offsetAlignment, builder);
+      auto valueToAlign =
+          builder.createOrFold<arith::AddIOp>(loc, offset, sliceSize);
+      offset = builder.createOrFold<IREE::Util::AlignOp>(loc, valueToAlign,
+                                                         offsetAlignment);
     }
 
-    return IREE::Util::align(loc, offset, rangeAlignment, builder);
+    return builder.createOrFold<IREE::Util::AlignOp>(
+          loc, offset, rangeAlignment);
```

Comment:
clang-format has a few complaints in here
https://github.com/google/iree/runs/4111833017?check_suite_focus=true